server_admin/topics/login-settings/forgot-password.adoc

```diff
@@ -1,34 +1,40 @@
 
 === Forgot Password
 
-If you enable it, users are able to reset their credentials if they forget their password or lose their OTP generator.
-Go to the `Realm Settings` left menu item, and click on the `Login` tab.  Switch on the `Forgot Password` switch.
+If you enable `Forgot Password`, users can reset their login credentials if they forget their password or lose their OTP generator. 
 
+.Procedure
+. Click *Realm Settings* in the menu.
+. Click the *Login* tab.
++
 .Login Tab
 image:{project_images}/login-tab.png[]
++
+. Toggle the *Forgot Password* switch to *ON*.
 
-A `forgot password` link will now show up on your login pages.
+A `forgot password` link displays in your login pages.
 
 .Forgot Password Link
 image:{project_images}/forgot-password-link.png[]
 
-Clicking on this link will bring the user
-to a page where they can enter in their username or email and receive an email with a link to reset their credentials.
+Click on this link to bring users where they can enter their username or email address and receive an email with a link to reset their credentials.
 
 .Forgot Password Page
 image:{project_images}/forgot-password-page.png[]
 
-The text sent in the email is completely configurable. You just need to extend or edit the theme associated with it.
-See the link:{developerguide_link}[{developerguide_name}] for more information.
+The text sent in the email is configurable. See link:{developerguide_link}[{developerguide_name}] for more information.
 
-When the user clicks on the email link, they will be asked to update their password, and, if they have an OTP generator
-set up, they will also be asked to reconfigure this as well.  Depending on the security requirements of your organization
-you may not want users to be able to reset their OTP generator through email.  You can change this behavior by
-going to the `Authentication` left menu item, clicking on the `Flows` tab, and selecting the `Reset Credentials` flow:
+When users click the email link, {project_name} asks them to update their password, and if they have set up an OTP generator, {project_name} asks them to reconfigure the OTP generator.  Depending on your orgazizations security requirements, you may not want users to reset their OTP generator through email. 
 
+To change this behavior:
+
+.Procedure
+. Click *Authentication* in the menu
+. Click the *Flows* tab.
+. Select *Reset Credentials* from the drop-down menu.
```

Comment:
See IBM guidelines for Drop-Down menu.

https://docs.engineering.redhat.com/display/CCSENABLE/GUI+elements

Reply:
Kinda awkward to apply the IBM standard since the drop-down has no label.

Reply:
Agreed.  Lets talk at our synchup

Reply:
Changing to 

`Select the *Reset Credentials* flow.`